🐛 Keycloak and quickstart updates

dmihalcik-virtru opened the pull request:
Some small updates for quickstart and related files:

- Updates KC parameters to match current set in backend umbrella chart. This fixes http://localhost:65432/auth/realms/tdf/.well-known/openid-configuration having some urls with the port and some without , among other things.
- Enable custom ingress port forward with the `OPENTDF_INGRESS_HOST_PORT` parameter. This is useful for local development, e.g. running a custom abacus or local demo app in the host OS  which is may be faster than running the dev app in the k8s cluster
- Adds `OPENTDF_LOAD_FRONTEND` boolean parameter to allow disabling abacus. Again, this is most useful while developing abacus itself (or other front end example apps)
- Renames `HELM_VALUES_PATH` `OPENTDF_HELM_VALUES_PATH` to match other env variable config names

elizabethhealy commented on quickstart/helm/secrets/templates/secrets.yaml:
```diff
@@ -83,8 +83,6 @@ type: Opaque
   KEYCLOAK_ADMIN: {{ "keycloakadmin" | b64enc | quote  }}
   KEYCLOAK_ADMIN_PASSWORD: {{ "mykeycloakpassword" | b64enc | quote }}
-  KC_HOSTNAME: {{ "localhost:65432" | b64enc | quote }}
-  KC_HOSTNAME_ADMIN: {{ "localhost:65432" | b64enc | quote }}
```
same question as backend PR

elizabethhealy reviewed (dismissed):
approved but cassandra asked to not merge anything before the release https://virtru.slack.com/archives/GNFF2JBQ9/p1674238473191969

dmihalcik-virtru commented:
I've rewritten this heavily, and added a couple of features (disabling abacus, allowing port-forward port selection) that I need for developing the test app

More context: https://github.com/opentdf/client-web/pull/138
